test/runnable/future.d

```diff
@@ -0,0 +1,43 @@
+/* PERMUTE_ARGS:
+ */
+
+class A
+{
+    @__future char msg() { return 'a'; }
+}
+
+class B : A
+{
+    char msg() { return 'b'; }
```

Comment:
Is there any way to check if this is producing the deprecation message? Otherwise this is only testing that override doesn't give an error when done over a `@__future` symbol.